Doc/library/re.rst

```diff
@@ -639,9 +639,9 @@ form.
 
 .. class:: RegexFlag
 
-   An :class:`~enum.Enum` class containing the regex options listed below.
+   An :class:`enum.IntFlag` class containing the regex options listed below.
 
-   .. versionadded:: 3.6 (documented in 3.11)
+   .. versionadded:: added to ``__all__`` in 3.11)
```

Comment:
Missing a parenthesis, so lets change to:

    .. versionadded:: 3.11 - added to ``__all__``